src/app/Main/Body/QueryPage/Results/TableResults/TableContent-Table.component.tsx

```diff
@@ -44,52 +36,36 @@ const useStyles = makeStyles({
 export interface TableProps {
     tableData: Record<string, string>[];
     columnAttributes: string[];
-    getSize: (
-        node: HTMLElement | null,
-        key: "height" | "totalWidth" | "partWidth"
-    ) => void;
-    isRowNotCol: boolean;
     isRowAndCol: boolean;
 }
 
 /**
  * @desc Returns TableContainer for the results
  * @param {Record<string, string>[]} allIsolates - list of objects with the counted isolates
  * @param {string[]} columnAttributes - column attributes for the table header
- * @param {(node: HTMLElement | null, key: "height" | "totalWidth" | "partWidth") => void} getSize - callback function to get the size of the header for the position of the main header
- * @param {boolean} isRowNotCol - true if row and no column is selected

```

Comment:
same here: these are descriptions of the props interface

Reply:
Done.